pkg/trace/config/apply.go

```diff
@@ -173,9 +173,23 @@ func (c *AgentConfig) applyDatadogConfig() error {
 	if config.Datadog.IsSet("apm_config.log_file") {
 		c.LogFilePath = config.Datadog.GetString("apm_config.log_file")
 	}
+
 	if config.Datadog.IsSet("apm_config.env") {
 		c.DefaultEnv = config.Datadog.GetString("apm_config.env")
+		log.Debugf("setting DefaultEnv to '%s' (from apm_config.env)", c.DefaultEnv)
```

Comment:
I think @mfpierre made it a "convention" to always capitalise log messages. Can we do that please?
```suggestion
		log.Debugf("Setting DefaultEnv to %q (from apm_config.env)", c.DefaultEnv)
```